### PR TITLE
Fix presetcli

### PR DESCRIPTION
**Pull request by stu-gott**
It turns out the VirtualMachineInstancePreset interface was never added to the KubevirtClient. So even though the code was present, it effectively never existed.

In addition to fixing that, this PR cleans up object names to match current conventions, and implements missing unit tests.

```release-note
Add VirtualMachineInstancePresets to kubecli
```

### CODE REVIEW

File: pkg/kubecli/vmipreset_test.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	It("should detect non existent VMIPresets", func() {
+		server.AppendHandlers(ghttp.CombineHandlers(
+			ghttp.VerifyRequest("GET", presetPath),
+			ghttp.RespondWithJSONEncoded(http.StatusNotFound, errors.NewNotFound(schema.GroupResource{}, "testpreset")),
+		))
+		_, err := client.VirtualMachineInstancePreset(k8sv1.NamespaceDefault).Get("testpreset", k8smetav1.GetOptions{})
+
+		Expect(server.ReceivedRequests()).To(HaveLen(1))
+		Expect(err).To(HaveOccurred())
+		Expect(errors.IsNotFound(err)).To(BeTrue())
```
Comment (petrkotas) [inline pkg/kubecli/vmipreset_test.go]: In case of failure, this expect would say: Expected false to be true. Would you please add description here?

**Comment by stu-gott** [comment]
ci test please

**Comment by davidvossel** [comment]
ci test please

**Comment by stu-gott** [comment]
ci test please

**Pull request merged**